package.json

```diff
@@ -57,17 +57,18 @@
     "gatsby": "^3.14.4",
     "prettier": "2.0.5",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react-dom": "^17.0.2",
+    "webpack": "^5.64.4"
   },
   "peerDependencies": {
     "gatsby": "^3.14.4",
     "react": "^17.0.2",
     "react-dom": "^17.0.2"
   },
   "scripts": {
-    "dev": "yarn develop",
+    "dev": "npm run develop",
     "build": "gatsby build",
-    "develop": "gatsby develop -p 7000",
+    "develop": "gatsby develop -p 7001",
```

Comment:
Running in port 7000 breaks CORS authentication with `home-api-medusa` on my end.

Reply:
That's because we have a CORS url configured here: https://github.com/lambda-curry/nibll-home/blob/master/apps/home/api-medusa/medusa-config.js#L2

Reply:
So should I undo this change, and then modify my `.env` file to point to port 7000?

Reply:
went ahead and answered my own question lol, this is resolved!